doc/man-sections/client-options.rst

```diff
@@ -87,6 +87,20 @@ configuration.
   The server configuration must specify an ``--auth-user-pass-verify``
   script to verify the username/password provided by the client.
 
+--external-auth
+  For security reasons, OpenVPN requires client-side credentials such as
+  client certificates or a username/password combination. The OpenVPN server
+  has the capability to delegate authentication to external systems using the
+  WEBAUTH protocol. In such cases, client credentials may be omitted.
```

Comment:
This description is a bit vague. Add a first sentence saying what the option does: e.g.,  "This client-only option indicates that user authentication options in the client configuration are not mandatory." Then follow with the above explanation?